.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm run lint
+npm run eslint
```

Comment:
This should go in ```.lintstagedrc```

Reply:
I have removed line 5 in the .husky/pre-commit file and updated the .lintstagedrc file